mailmerge/__main__.py

```diff
@@ -71,13 +71,13 @@
     "--config", "config_path",
     default="mailmerge_server.conf",
     type=click.Path(),
-    help="server configuration (mailmerge_server.conf)",
+    help="SMTP configuration (mailmerge_server.conf)",
```

Comment:
```suggestion
    help="SMTP server configuration (mailmerge_server.conf)",
```

Reply:
That makes the `--help` output wrap for me, which is why I picked the shorter version:
```
  --database PATH                 database CSV (mailmerge_database.csv)
  --config PATH                   SMTP server configuration
                                  (mailmerge_server.conf)

  --output-format [colorized|text|raw]
```

vs 
```
  --config PATH                   SMTP configuration (mailmerge_server.conf)
```

Reply:
sure that's fine